### DISCUSSION TITLE
hertzbeatl软件用1panle的初次配置及安装的实用教程

### DISCUSSION THREAD

**Discussion by xiemeng00** [1Panel-dev/1Panel#5586] [Q&A]
### 1Panel 版本

v1.10.1-lts

### 请描述您的需求或者改进建议

由于安装时不懂如何配置时序数据库及Mysql,以及如何配置compose 及一些基本设置，网上只有直接docker安装方式，初学1panel请大佬出个安装配置使用教程。官网也只介绍如何DOCKER安装，并没有1panel安装方法。请多出点1panel详细视频。方便大家使用并支持。感谢

### 请描述你建议的实现方案

![_2{A2$%8H7S6YXR06JLXZX7](https://github.com/1Panel-dev/1Panel/assets/167733485/e08eef82-ea44-4673-8b95-d6424516e823)
希望在B站做些详细每一步配置操作视频

**Comment by wanghe-fit2cloud** [comment]
[第三方商店](https://github.com/okxlin/appstore/tree/localApps/apps/hertzbeat) 中有 `hertzbeat` 这款应用，具体使用教程可以参考论坛置顶文章 [1Panel 本地应用创建技巧及第三方应用库举例](https://bbs.fit2cloud.com/t/topic/640/)
